docs/docs/capella/base.md

```diff
@@ -149,12 +149,13 @@ Ubuntu `focal` repository (<https://packages.ubuntu.com/focal/libwebkit2gtk-4.0-
 
 First of all, you have to add the source to your `apt`-sources and add the apt keys.
 
-Recommendation: Spawn a Docker container and execute the steps inside the container.
+Recommendation: Spawn a `debian:bookworm` Docker container and execute the steps inside the container.
```

Comment:
It's still referencing bullseye on line 147.

Does it still make sense to have those instructions? Why would one download an older version of `libwebkit2gtk`?

Reply:
The Capella description editor is not compatible with the latest version of `libwebkit2gtk`. The description editor doesn't load. I tried it with the latest Capella version with bookworm and had the hope that we don't need to downgrade it anymore, but this was not the case.